sdk/eventhubs/azure_messaging_eventhubs/examples/richards_test.rs

```diff
@@ -0,0 +1,98 @@
+use core::f32;
```

Comment:
1. Copyright header.
2. Let's not use people's names in code. Probably not exactly PII, but feels close enough. Also raises more questions than answers.
3. If it's a test, why is it not in the `tests/` directory as an integration test?

Reply:
...even if live-only.

Reply:
It was an oops - this shouldn't have been checked in.